src/modules/coin/resources.rs

```diff
@@ -14,7 +14,7 @@ use crate::{db_helpers, errors, modules, types};
 /// for the given timestamp/block_height.
 pub async fn get_near_balance(
     pool: web::Data<sqlx::Pool<sqlx::Postgres>>,
-    request: web::Path<schemas::BalanceRequest>,
+    request: actix_web_validator::Path<schemas::BalanceRequest>,
```

Comment:
Could you please make the same thing for all the endpoints? You need to change this file and `nft/resources.rs`

Reply:
Done:)